submit-web/src/components/Shared/layout/Header/EAOAppBar.tsx

```diff
@@ -14,29 +14,31 @@ export default function EAOAppBar() {
         position="static"
         color="inherit"
         sx={{
-          borderBottom: `1px solid ${theme.palette.grey[300]}`,
+          borderBottom: `1px solid #d8d8d8`,
```

Comment:
how can we mitigate using a custom color like this, can we explore using a color that exists in BCDesignTokens?

Reply:
i looked through the bcdesign tokens i did not find d8d8d8

Reply:
then this should prompt a conversation. Let's talk with MA about only using BCDesignTokens